sdk/containerregistry/container-registry/samples/v1/typescript/package.json

```diff
@@ -32,7 +32,7 @@
     "@azure/identity": "^2.0.1"
   },
   "devDependencies": {
-    "typescript": "~4.2.0",
+    "typescript": "~4.4.0",
```

Comment:
@praveenkuttappan do you know why this version is changed?

Reply:
dev-tool to update samples made this change.

Reply:
@witemple-msft explained that the dev-tool is using the version of typescript from dev-tool's deps. There's a concern that samples may not work with newer version of typescript.  I am reverting this change.